go/kbfs/libkbfs/md_ops.go

```diff
@@ -26,7 +26,10 @@ import (
 )
 
 const (
-	maxAllowedMerkleGapServer = 13 * time.Hour
+	// Update 2024-02-16: we decided to disable the merkel gap check (by
+	// increase the allowed gap to a very high number) since mdmerkle is slow
+	// and this gap check is doing mroe harm than good.
+	maxAllowedMerkleGapServer = 200 * (time.Hour * 24 * 365)
 	// Our contract with the server states that it won't accept KBFS
 	// writes if more than 13 hours have passed since the last Merkle
```

Comment:
"13 hours" -> "200 years" and maybe remove the TODO below as well?  I guess this whole comment is kind of silly now, maybe it can just be removed.